cypress/commons/utils/apiUtils.js

```diff
@@ -390,6 +409,15 @@ const interceptUpdateDataset = (options) => {
   return alias;
 };
 
+const interceptDeleteDataset = (datasetName) => {
+  const alias = forgeAlias('reqDeleteDataset');
+  cy.intercept({ method: 'DELETE', url: API_REGEX.DATASET, times: 1 }, (req) => {
+    if (stub.isEnabledFor('GET_DATASETS')) stub.deleteScenarioByName(datasetName);
```

Comment:
`delete*Scenario*ByName` : this is (probably) not the method you're looking for :sweat_smile: 

Also, the condition should also include ` || stub.isEnabledFor('DELETE_DATASET')` to remove the fake dataset from the data in the `stubbing` class (as mentioned in a previous comment, we can probably use `CREATE_DATASET` instead of creating a new type `DELETE_DATASET`)

Reply:
The test might not try to access the stubbed data again. As long as the stubbed "delete" request returns a positive answer, the front-end believes that the dataset is removed :shrug: